src/lib/spinel/multi_frame_buffer.hpp

```diff
@@ -40,9 +40,14 @@
 
 #include <openthread/error.h>
 
+#include "lib/utils/endian.hpp"
+
 namespace ot {
 namespace Spinel {
 
+using Lib::Utils::LittleEndian::ReadUint16;
+using Lib::Utils::LittleEndian::WriteUint16;
+
```

Comment:
It is not a good practice to write the namespace `using` in a header file. You can refer to: https://www.oreilly.com/library/view/c-coding-standards/0321113586/ch60.html#:~:text=Summary,explicitly%20namespace%2Dqualify%20all%20names.

Reply:
I agree. It's generally better to avoid `using` directives in header files.

I suggest using `LittleEndian` and `BigEndian` directly in the code to make the endianness handling explicit and easier to understand, and reducing the chance of unintentional behavior.

Reply:
Accept. Thanks!